src/Nancy.Templates/Nancy.Templates/Nancy.Templates.csproj

```diff
@@ -15,7 +15,7 @@
     <AssemblyOriginatorKeyFile>Key.snk</AssemblyOriginatorKeyFile>
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
-    <MinimumVisualStudioVersion>12.0</MinimumVisualStudioVersion>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>

```

Comment:
Yes, this shouldn't have gone in. Just got into a bit of a trouble on the git front when forgetting to check out a new branch and then pushing commits. 

My original intention was to submit a pull request containing only the change in 9b3f1d7. So i will try and get it back to this point.

Thanks for the feedback and apologies for making this one more complex than it should be.